extensions/guacamole-auth-duo/src/main/java/org/apache/guacamole/auth/duo/form/DuoSignedResponseField.java

```diff
@@ -81,8 +80,7 @@ public DuoSignedResponseField(String apiHost, String signedRequest) {
      * @return
      *     The hostname of the DuoWeb API endpoint.
      */
-    @JsonProperty("apiHost")
-    public String getAPIHost() {
+    public String getApiHost() {
```

Comment:
I see this changed, here, to a more strict CamelCase, along with a couple of others like it in other places in the code, but I don't see that any of the calls to this method get changed. Am I missing something?

Reply:
Nope - there aren't any such calls. This method serves only to expose data for serialization into JSON by Jackson. Renaming the function in this case allows the JSON property to be `apiHost` without requiring a Jackson-specific annotation to force that name.